content/docs/command-reference/exp/pull.md

```diff
@@ -0,0 +1,65 @@
+# exp pull
+
+Download a single [experiment](/doc/command-reference/exp) from a Git remote, and
+its data from a `dvc remote`.
+
+## Synopsis
+
+```usage
+usage: dvc experiments pull [-h] [-q | -v] [-f] [--no-cache]
+                            [-r <name>] [-j <number>] [--run-cache]
+                            git_remote experiment
+
+positional arguments:
+  git_remote          Git remote name or repo URL
+  experiment          Experiment to pull
+```
+
+## Description
+
+The `dvc exp push` and `dvc exp pull` commands are the means for sharing
+experiments across <abbr>repository</abbr> copies via Git (and DVC) remotes.
+
+> Plain `git push` and `git pull` don't work with `dvc experiments` because
+> these are saved under custom Git references. See **How does DVC track
+> experiments?** in `dvc exp run` to learn more about DVC experiment storage.
+
+A working `git_remote` name (e.g. `origin`) or valid Git repo's URL is required,
+as well as a single `experiment` name or hash (see `dvc exp run`) to pull.
+
+By default, this command will also try to [pull](/doc/command-reference/pull)
+all <abbr>cached</abbr> data associated with the experiment to DVC
+[remote storage](/doc/command-reference/remote), unless `--no-cache` is used.
+The default remote is used (see `dvc remote default`) unless a specific one is
+given with `--remote`.
+
+> 💡 Note that `git push <git_remote> --delete <experiment>` can be used to
+> delete a pushed experiment.
+
+## Options
+
+- `-f`, `--force` - rewrite the `experiment` commit if it already exists in the
+  local repo. Equivalent to `git push --force` (rewrites history)
+
+- `--no-cache` - do not pull cached files files associated with this experiment
+  from DVC remote storage.
+
+- `-r <name>`, `--remote <name>` - name of the `dvc remote` to pull cached files
+  to.
```

Comment:
Note for self: should we consider renaming here to `--dvc-remote` to disambiguate? Rn we have

`dvc exp pu* <remote> <experiment> --remote <remote>`

Reply:
I think the current core repo CLI usage explicitly uses `git_remote` everywhere that we want git remotes, and `remote` on its own is always assumed to mean DVC remote.

Reply:
You're right, this shouldn't be confusing. Resolving